.github/workflows/main.yml

```diff
@@ -139,7 +139,7 @@ jobs:
           mkdir myextension
           python -m copier copy -l -d author_name="My Name" -d test=n -d repository="https://github.com/test/lab-extension" --vcs-ref HEAD --UNSAFE . myextension
           pushd myextension
-          pip install "jupyterlab>=4.0.0,<5"
+          pip install "jupyterlab>=4.0.0,<5" setuptools
```

Comment:
@fcollonval What was the reason for explicitly installing `setuptools` here?

Reply:
The issue is that setuptools is not installed by default with Python. This was breaking the JupyterLab cli:

https://github.com/jupyterlab/jupyterlab/blob/21ffea9749637633b4a4d82b3f277b55f80afb3d/jupyterlab/federated_labextensions.py#L464

Reply:
OK thanks. I was wondering if this could also be related to https://github.com/jupyterlab/jupyterlab/issues/16404